easybuild/easyconfigs/n/Net-core/Net-core-2.1.8.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'Tarball'
+
+name = 'Net-core'
+version = '2.1.8'
+
+homepage = 'https://dotnet.microsoft.com/'
+description = """.NET Core is a free and open-source managed computer software framework for the Windows,
+ Linux, and macOS operating systems .NET Core fully supports C# and F# and partially supports Visual Basic"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [
+    'https://download.visualstudio.microsoft.com/download/pr/927cbf41-157f-488b-ac7d-32034defebc3/41b011418f795d5a76bf930ad57a055b/'
```

Comment:
You could put that url into a variable, which you create in two steps, or do
'https://download.visualstudio.microsoft.com/download/pr/' +
'927cbf41-157f-488b-ac7d-32034defebc3/41b011418f795d5a76bf930ad57a055b/'
which should? be ok according to pep8